asm/code_80428A0.s

```diff
@@ -1368,7 +1368,7 @@ _08043594:
 	bl sub_806B6C4
 	b _080435D4
 	.align 2, 0
-_080435B4: .4byte gUnknown_80F5668
+_080435B4: .4byte gCurrentMusic
```

Comment:
```suggestion
_080435B4: .4byte gDungeonBGMusic
```

Feel like that is more descriptive given it's an array of each BG Music # corresponded to each dungeon

Reply:
I went for `gDungeonMusic`.